libs/payments/stripe/project.json

```diff
@@ -24,7 +24,7 @@
         ]
       }
     },
-    "test-unit": {
+    "test-integration": {
```

Comment:
Cool! But we should really move to having unit run all normal .spec.ts files, and have .int.spec.ts files run via test-integration.

Reply:
For posterity - we discussed that since this change will also need to be applied to payments-cart, payments-paypal, and shared-account-account, we will make this update in a separate PR.